Library/Homebrew/shims/scm/git

```diff
@@ -28,9 +34,7 @@ realpath() {
     else
       path="$dir/$dest"
     fi
```

Comment:
If you wanted to reuse `abspath` in `realpath`, the above `if` is what you would need to refactor (this is also essentially my suggested `abspath` implementation).